src/index.html

```diff
@@ -6,9 +6,41 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="#home">
+          <img
+            src="./images/logo.png"
+            alt="logo"
+            class="logo__img"
+          >
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a href="#home" class="nav__link is-active">Apple</a></li>
+          <li class="nav__item"><a href="#home" class="nav__link">Samsung</a></li>
+          <li class="nav__item"><a href="#home" class="nav__link">Smartphones</a></li>
+          <li class="nav__item">
```

Comment:
set corresponding value for href for all links, not home for all of them
and check indents
```suggestion
          <li class="nav__item">
            <a href="#apple" class="nav__link is-active">Apple</a>
          </li>
          <li class="nav__item">
            <a href="#samsung" class="nav__link">Samsung</a>
          </li>
          <li class="nav__item">
            <a href="#home" class="nav__link">Smartphones</a>
          </li>
          <li class="nav__item">
```